WooCommerce/StoreWidgets/StoreInfoProvider.swift

```diff
@@ -59,19 +64,33 @@ struct StoreInfoProvider: TimelineProvider {
     /// TODO: Update with real data.
     ///
     func getTimeline(in context: Context, completion: @escaping (Timeline<StoreInfoEntry>) -> Void) {
-        // TODO: Temp store name to check dependency status while we fetch real data.
-        let dependencies = Self.fetchDependencies()
-        let authStatus = dependencies?.authToken != nil ? "Authenticated" : "Non Authenticated"
-        let storeName = dependencies?.storeName ?? "Undefined Shop"
-        let entry = StoreInfoEntry(date: Date(),
-                                   range: "Today",
-                                   name: "\(authStatus) - \(storeName)",
-                                   revenue: "$132.234",
-                                   visitors: "67",
-                                   orders: "23",
-                                   conversion: "37%")
-        let timeline = Timeline<StoreInfoEntry>(entries: [entry], policy: .never)
-        completion(timeline)
+        guard let dependencies = Self.fetchDependencies() else {
+            return // TODO: Dispatch non auth error entry
+        }
+
+        let strongService = StoreInfoDataService(authToken: dependencies.authToken)
+        networkService = strongService
+        Task {
+            do {
+                let todayStats = try await strongService.fetchTodayStats(for: dependencies.storeID)
+
+                // TODO: Use proper store formatting.
+                let entry = StoreInfoEntry(date: Date(),
+                                           range: Localization.today,
+                                           name: dependencies.storeName,
+                                           revenue: "$\(todayStats.revenue)",
+                                           visitors: "\(todayStats.totalVisitors)",
+                                           orders: "\(todayStats.totalOrders)",
+                                           conversion: "\(todayStats.conversion)%")
+                let timeline = Timeline<StoreInfoEntry>(entries: [entry], policy: .never)
```

Comment:
Should be something else than `.never` to request another update in the future.
Not a requirement for this PR.

Reply:
Was aiming to do it later, but since it's short I did it now.

https://github.com/woocommerce/woocommerce-ios/pull/7690/commits/ac412dc6289aa6faf21d2919b8c7ce13c6a585b1